pkg/webui/console/views/device-general-settings/network-server-form/index.js

```diff
@@ -245,6 +245,10 @@ const NetworkServerForm = React.memo(props => {
         context: validationContext,
         stripUnknown: true,
       })
+      // Make sure `ping_slot_frequency` makes it into patch.
+      if (values.mac_settings.ping_slot_frequency) {
+        castedValues.mac_settings.ping_slot_frequency = values.mac_settings.ping_slot_frequency
+      }
```

Comment:
This should be solved via the validation schema in `/console/views/device-general-settings/network-server-form/validation-schema.js`